config/default.yml

```diff
@@ -29,30 +30,50 @@ Metrics/MethodLength:
     - api/sinatra/**/*
 
 Metrics/AbcSize:
+  Max: 18
+  Severity: refactor
   Exclude:
     - api/sinatra/**/*
-
+    
+Metrics/CyclomaticComplexity:
+  Severity: refactor
+  
 Metrics/BlockLength:
   Exclude:
     - '*.gemspec'
     - '**/*.rake'
-    - 'spec/**/*.rb'
+    - 'config/initialize/*'
+    - 'spec/**/*'
+    - 'api/**/*'
+    
+Metrics/ClassLength:
+  Exclude:
+    - spec/**/*
```

Comment:
I think we should remove this exclusion of specs. Once a test hits max class length, it usually makes sense to split it (and it's even easier than with actual classes)

Reply:
This is not a new rule - I just moved them around. Do normal specs even usually trigger this cop. they are not classes. This directive is I think mostly for unimportant test classes.